project01/paulo/apple.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <cinder/gl/gl.h>
+
+class GameCanvas;
+
+class Apple
+{
+public:
+  explicit Apple(GameCanvas&);
+  ~Apple();
```

Comment:
if not needed, dont define it. it is not a good practice, as it inhibits move ctor/assignment op, see here A can move, but AA cant:

```
struct A { std::unique_ptr<std::string> foo; };
struct AA { ~AA() {} std::unique_ptr<std::string> foo; };
```